searx/engines/duckduckgo_extra.py

```diff
@@ -69,28 +73,68 @@ def request(query, params):
         args['p'] = safe_search  # "-1", "1"
 
     logger.debug("cookies: %s", params['cookies'])
+
+    path = 'i'
+    if ddg_category == 'videos':
+        path = 'v'
+    elif ddg_category == 'news':
+        path = 'news'
     args = urlencode(args)
-    params['url'] = 'https://duckduckgo.com/i.js?{args}'.format(args=args)
+    params['url'] = f'https://duckduckgo.com/{path}.js?{args}'
 
     return params
 
 
+def _image_result(result):
+    return {
+        'template': 'images.html',
+        'url': result['url'],
+        'title': result['title'],
+        'content': '',
+        'thumbnail_src': result['thumbnail'],
+        'img_src': result['image'],
+        'img_format': '%s x %s' % (result['width'], result['height']),
+        'source': result['source'],
+    }
+
+
+def _video_result(result):
+    return {
+        'template': 'videos.html',
+        'url': result['content'],
+        'title': result['title'],
+        'content': result['description'],
+        'thumbnail': result['images'].get('small') or result['images'].get('medium'),
+        'iframe_src': result['embed_url'],
+        'source': result['provider'],
+        'length': result['duration'],
+        'metadata': result.get('uploader'),
+    }
+
+
+def _news_result(result):
+    return {
+        'url': result['url'],
+        'title': result['title'],
+        'content': result['excerpt'],
+        'img_src': result.get('image'),
```

Comment:
The images in the news category are often very big images and not small thumbs.  Since SearXNG uses this image as a thumb, I think it might be better we remove `img_src` the result items of ddg-news.

Reply:
Okay, fine with me :+1:

Reply:
(removed)